test_common/test_config_register.py

```diff
@@ -11,9 +11,10 @@ def test_config_register():
     ADDR_VALUE = 4
 
     # Check that compilation to CoreIR works. Delete JSON file afterwards.
-    cr = DefineConfigRegister(WIDTH, m.bits(ADDR_VALUE, ADDR_WIDTH), True)
+    cr = define_config_register(WIDTH, m.bits(ADDR_VALUE, ADDR_WIDTH), True)
     m.compile("config_register", cr, output='coreir')
-    os.system("config_register.json")
+    res = os.system("\\rm config_register.json")
```

Comment:
I think we should start regressing the output files against golds so we know when they change (either from an internal change or a dependency change such as in magma). For now we could just keep a directory of `gold` files and use `diff` to check they are the same.

Reply:
Done.